plugins/obs-ffmpeg/obs-ffmpeg-video-encoders.c

```diff
@@ -256,7 +256,7 @@ bool ffmpeg_video_encode(struct ffmpeg_video_encoder *enc,
 		packet->keyframe = !!(av_pkt.flags & AV_PKT_FLAG_KEY);
 		*received_packet = true;
 
-		uint64_t recv_ts_nsec =
+		const int64_t recv_ts_nsec =
 			util_mul_div64((uint64_t)av_pkt.pts,
```

Comment:
The function util_mul_div64 returns a uint64_t. Won't this warn about possible loss of data converting from uint to int?

Reply:
MSVC is fine with out default settings. Is our GCC set up to use -Wsign-conversion? It's not even part of -Wall. I guess I can just add the cast in case.

Reply:
The newly added cast is probably fine. Resolve at your discretion.